src/pages/editor/OrganizationOverview.tsx

```diff
@@ -0,0 +1,150 @@
+    staleTime: Infinity,
+    cacheTime: 1_800_000, // 30 minutes
+    meta: { errorMessage: t('feedback.error.get_institution') },
+  });
+
+  const allSubUnits = getSortedSubUnits(organizationQuery.data?.hasPart);
+
+  return (
+    <>
+      <Helmet>
+        <title>{t('editor.organization_overview')}</title>
+      </Helmet>
+      <Typography variant="h1" sx={{ mb: '1rem' }}>
+        {t('editor.organization_overview')}
+      </Typography>
+
+      <Typography sx={{ mb: '2rem' }}>
+        <Trans t={t} i18nKey="editor.institution.institution_helper_text">
+          <Link href="mailto:kontakt@sikt.no" target="_blank" rel="noopener noreferrer" />
+        </Trans>
+      </Typography>
+
+      {organizationQuery.isLoading ? (
+        <ListSkeleton height={100} minWidth={100} />
+      ) : (
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
+          <Autocomplete
+            data-testid={dataTestId.editor.organizationOverviewSearchField}
+            options={allSubUnits}
+            inputMode="search"
+            getOptionLabel={(option) => getLanguageString(option.labels)}
+            getOptionKey={(option) => option.id}
+            onChange={(_, selectedUnit) => setSearchId(selectedUnit?.id ?? '')}
+            renderInput={(params) => (
+              <TextField {...params} variant="outlined" label={t('search.search_for_sub_unit')} />
+            )}
+          />
+
+          {organizationQuery.data?.hasPart?.map((organization) => (
+            <OrganizationAccordion key={organization.id} organization={organization} searchId={searchId} />
+          ))}
+        </Box>
+      )}
+    </>
+  );
+};
+
+interface OrganizationAccordionProps {
+  organization: Organization;
+  searchId: string;
+  includeAllSubunits?: boolean;
+  level?: number;
+}
+
+const OrganizationAccordion = ({
+  organization,
+  searchId,
+  level = 0,
+  includeAllSubunits = false,
+}: OrganizationAccordionProps) => {
+  const { t } = useTranslation();
+
+  const [expanded, setExpanded] = useState(false);
+
+  const isSearchedUnit = organization.id === searchId;
+
+  if (!!searchId && !isSearchedUnit && !includeAllSubunits) {
+    const allSubunits = getAllChildOrganizations(organization.hasPart);
+    if (!allSubunits.some((subunit) => subunit.id === searchId)) {
+      return null; // Hide this element if the searched ID is not a part of this unit
+    }
+  }
+
+  const subunitsCount = organization.hasPart?.length ?? 0;
+
+  return (
+    <Accordion
+      data-testid={dataTestId.editor.organizationAccordion(organization.id)}
+      elevation={2}
+      disableGutters
+      sx={{ bgcolor: level % 2 === 0 ? 'secondary.main' : 'secondary.light', ml: { xs: undefined, md: `${level}rem` } }}
+      expanded={expanded || (!!searchId && !includeAllSubunits)}
+      onChange={() => setExpanded(!expanded)}>
+      <AccordionSummary expandIcon={<ExpandMoreIcon sx={{ visibility: subunitsCount > 0 ? null : 'hidden' }} />}>
```

Comment:
Av en eller annen grunn irriterer det meg veldig at det kommer en 'pointer' selv om man ikke kan expande accordion😅 Men finnger ingen god løsning på det, da det virker som om `Accordion` har en innebygd hover-effekt som ikke kan overrides av `cursor`.

Reply:
Enig, egnetlig burde vi rendret bare `<Paper>` om det ikke er noen subunits, og `Accordion` bare når det er noe som faktisk kan expandes. Starta litt med det, men fant ut at det vare en del ekstra styling på Accordion som ikke var litt herk å reprodusere også for Paper :/

En annen svakhet som det er nå er at man ikke kan markere teksten som står i en Accordion, så ikke noe kjempefan av kompnenten(e) der egentlig..

Reply:
Nå har jeg en teit hack hvor ikonet bare skjules om den ikke er klikkbar også, så litt flere ting som hadde vært bedre om ikke alt ble presset inn i en accordion

Reply:
Blir nesten helt likt med dette ser jeg:
```
<Paper
  elevation={2}
  sx={{
    p: '1rem',
    bgcolor: level % 2 === 0 ? 'secondary.main' : 'secondary.light',
    ml: { xs: undefined, md: `${level}rem` },
  }}>
```

Men det blir litt annet herk i ekspandering/søk osv, så tror jeg bare legger den litt på vent jeg altså 🤔